src/components/gauge/gauge.tsx

```diff
@@ -0,0 +1,445 @@
+import {
```

Comment:
Please take a look at some "magic" numbers in the code. I'd like these number to be constants, so be know the semantics behind the number.

Reply:
Those numbers are to take a percentage of a variable number. I do not know how I can explain the purpose of the numbers by assigning them a name.
Perhaps `const gaugeContainer80percent = 0.8;`?
It is really necessary to explain each number? It's just used in a formula to calculates the size to be responsive.

Reply:
Magic Numbers fixed in commits af4ba8a42626d1f525a9cb38ede6412ba416ddb2 beb727049f504fe25517716b0b1e6f789ef7442b ef4c6c284ffd4503c0216ca81a8c927ca297e085